lib/diagnostic.rb

```diff
@@ -43,6 +44,10 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    lines = File.open(ARGV[0]) do |file|
```

Comment:
ARGV[0] is a reference to the first argument to the method (similar to `arguments[0]` in JavaScript). No need to use it here, since we have a named first argument.